S/SPRAL/build_tarballs.jl

```diff
@@ -42,11 +52,11 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="METIS_jll", uuid="d00139f3-1899-568f-a2f0-47f597d42d70"))
-    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
-    Dependency(PackageSpec(name="OpenBLAS32_jll", uuid="656ef2d0-ae68-5445-9ca0-591084a874a2"))
```

Comment:
Ok, last question: the library uses LP64 interface, you must make sure that your consumer package will have at least one backing LP64 BLAS, otherwise you'll get runtime errors.  Would you do that?

Reply:
Yes, I recompiled SPRAL to use it as a linear solver of Ipopt, like MUMPS. And in Ipopt.jl, we are already taking care of that: https://github.com/jump-dev/Ipopt.jl/blob/master/src/Ipopt.jl#L15-L28